src/drivers/express/ExpressRouteDriver.ts

```diff
@@ -312,15 +311,6 @@ export default class ExpressRouteDriver {
         }),
       );
 
-    router.get(
```

Comment:
isn't this learning objects not library service?

Reply:
It's proxying to the CART_API which is why it's grouped in the library service controller.